tests/common/include/tests/common/energy_loss.inl

```diff
@@ -50,14 +50,19 @@ TEST_P(EnergyLossBetheValidation, bethe_energy_loss) {
         slab.path_segment(is) / slab.get_material().mass_density() /
         (unit<scalar>::MeV * unit<scalar>::cm2 / unit<scalar>::g)};
 
+    const scalar expected_dEdx = std::get<2>(GetParam());
+
     // Check if difference is within 5% error
-    EXPECT_TRUE(std::abs(std::get<2>(GetParam()) - dEdx) / dEdx < 0.05f);
+    EXPECT_NEAR((expected_dEdx - dEdx) / dEdx, 0.f, 0.05f);
 }
 
+/*
+@ NOTE: Test fails with H2liquid and 0.1003 GeV muons
```

Comment:
I disabled the test for this specific case as the difference turns out to be 18%

Reply:
OK I was fixing wrong test